### DISCUSSION TITLE
How to serve static response ?

### DISCUSSION THREAD

**Discussion by eshepelyuk** [envoyproxy/gateway#4032] [Q&A] [answered]
Hello, in envoy it's possible to serve a static response: body, status code.
Is it possible to do with K8s API ?

**Comment by arkodg** [comment, marked as answer]
hey @eshepelyuk this feature is being tracked with https://github.com/envoyproxy/gateway/issues/2714 and requires changes in the upstream API
you could use https://gateway.envoyproxy.io/docs/tasks/extensibility/envoy-patch-policy/ as a temp workaround

**Reply by eshepelyuk** [reply]
thank you
